### DISCUSSION TITLE
[BUG]1.2.9 No incoming peers anymore

### DISCUSSION THREAD

**Discussion by ZwaZo22** [Chia-Network/chia-blockchain#8830] [Support]
Since the 1.2.9 upgrade, my node doesn't get incoming peer connections anymore. I used to get about 50 nodes connected. Now only the 8 outgoing peer connections are shown with the chia show -c 

The 8444 tcp port is properly mapped on the router to the server.

**To Reproduce**

upgrade to 1.2.9

Launch Chia

then `chia show -c`

```
Connections:
Type      IP                                     Ports       NodeID      Last Connect      MiB Up|Dwn
FARMER    127.0.0.1                              57590/8447  0ce23453... Oct 05 16:20:27      0.4|0.0
FULL_NODE 206.174.40.98                           8444/8444  e0c1c607... Oct 06 00:37:22      2.5|3.2
                                                 -SB Height:   956968    -Hash: 131bda0e...
FULL_NODE 90.119.82.157                           8444/8444  9d75cb63... Oct 06 00:37:22      1.7|29.7
                                                 -SB Height:   956968    -Hash: 131bda0e...
FULL_NODE 46.236.231.75                           8444/8444  db8d4b21... Oct 06 00:37:22      1.3|16.5
                                                 -SB Height:   956968    -Hash: 131bda0e...
FULL_NODE 167.179.151.69                          8444/8444  50eaef3a... Oct 06 00:37:22      2.7|6.8
                                                 -SB Height:   956968    -Hash: 131bda0e...
FULL_NODE 46.153.130.194                          8444/8444  cd7f4875... Oct 06 00:37:22      1.8|11.9
                                                 -SB Height:   956968    -Hash: 131bda0e...
FULL_NODE 195.211.160.112                         8444/8444  8012dde2... Oct 06 00:37:22      2.5|13.6
                                                 -SB Height:   956968    -Hash: 131bda0e...
WALLET    127.0.0.1                              58216/8449  1b48fee9... Oct 06 00:37:07      6.3|0.0
FULL_NODE 196.234.245.102                         8444/8444  c5edcd84... Oct 06 00:37:22      2.1|6.2
                                                 -SB Height:   956968    -Hash: 131bda0e...
FULL_NODE 99.152.113.146                          8444/8444  7379e2bd... Oct 06 00:37:22      1.4|13.7
                                                 -SB Height:   956968    -Hash: 131bda0e...
```

**Expected behavior**

show all the incoming peers

**Desktop**

- OS: Ubuntu 20.04 lts / CLI

below the 8444 port monitoring from uptimerobot.com showing the port availability. The downtime shown on the graph was when upgrading the node + server reboot.

![Capture d’écran 2021-10-06 à 00 27 08](https://user-images.githubusercontent.com/47155095/136111610-807ca7a4-8f9d-4801-be2c-8c08dce4a140.jpg)

**Comment by loppefaaret** [comment]
ZwaZoo22 cought me on keybase (@zwazo) #support, and after a little talk - this might be a home ISP that is potentially causing this ? 
i manually connected just fine to his DDNS name, and hopefully, my peer will start gossiping his address out to other peers.
I'm suspecting an IP change on his home connection might have the chia network peers have his peer ID cached under a different IP, and that is causing the no incoming connections - we planned to do a follow up later on, on the situation of his peering status

**Comment by BrandtH22** [comment]
@ZwaZo22 , since this ticket was opened a number of years ago without further comments it seems this issue has been resolved. 
We will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions or comments.

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
